### PR TITLE
db.hackclub.com

### PR DESCRIPTION
**Pull request by Iamusicgab**
# Adding db.hackclub.com`

## Description

PR for adding db.hackclub.com. The subdomain will be used by Don Bosco Technical Institute of Makati for their Hack Club website.

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -837,6 +838,10 @@ daysofservice:
   ttl: 1
   type: CNAME
   value: cname.vercel-dns.com.
+db:
+  ttl: 1
```
Comment (Muirrum) [inline hackclub.com.yaml:842]:
```suggestion
  ttl: 600
```

**Review by Muirrum — CHANGES_REQUESTED** [review]
TTL needs to be bumped, otherwise LGTM!

**Review by grymmy — CHANGES_REQUESTED** [review]
I would request that another subdomain choice be used, not 'db'.

**Comment by grymmy** [comment]
landing this as your feedback seems to have been resolved @Muirrum

**Pull request merged**